alf/environments/make_penv.py

```diff
@@ -20,13 +20,14 @@ def gen_penv():
     current_path = os.path.abspath(__file__)
     dir_path = os.path.dirname(current_path)
     os.chdir(dir_path)
+
     # install pybind11 if it's not installed.
     try:
         import pybind11
     except:
         assert os.system(
             "pip install pybind11") == 0, "Fail to pip install pybind11"
-    python = f"python{sys.version_info.major}.{sys.version_info.minor}"
+    python = sys.executable
```

Comment:
What is the problem with the previous version of the code?
The new version does not work on my machine. On my machine, sys.executable get "python", but python-config does not exist.

Reply:
I think it is due to https://github.com/pypa/virtualenv/issues/169

Let me revert this change then. I found a way to have `python3.x` in my environment now.